src/Dependencies/xUnit.net/project.json

```diff
@@ -0,0 +1,10 @@
+{
+  "supports": {},
+  "dependencies": {
+    "xunit": "2.1.0",
+    "xunit.runner.console": "2.1.0"
```

Comment:
They don't produce any assemblies or references, solely used to for project.json inheritance. When NuGet supports that, these can be removed.